docs/components/radio.md

```diff
@@ -1,161 +1,161 @@
+    <div class="horizontal-container">
+      <KRadio name="test" :selected-value="true" v-model="radioValue">Boolean</KRadio>
+      <KRadio name="test" selected-value="string" v-model="radioValue">String</KRadio>
+      <KRadio name="test" :selected-value="objA" v-model="radioValue">Object A</KRadio>
+      <KRadio name="test" :selected-value="objB" v-model="radioValue">Object B</KRadio>
     </div>
   </template>
 </KCard>
 
-```html
+```vue
 <template>
-  <div>Selected: {{ radioGroup }}</div>
-  <KRadio name="test" :selected-value="true" v-model="radioGroup">Boolean</KRadio>
-  <KRadio name="test" selected-value="string" v-model="radioGroup">String</KRadio>
-  <KRadio name="test" :selected-value="objA" v-model="radioGroup">Object A</KRadio>
-  <KRadio name="test" :selected-value="objB" v-model="radioGroup">Object B</KRadio>
+  <div>Selected: {{ radioValue }}</div>
+  <KRadio name="test" :selected-value="true" v-model="radioValue">Boolean</KRadio>
+  <KRadio name="test" selected-value="string" v-model="radioValue">String</KRadio>
+  <KRadio name="test" :selected-value="objA" v-model="radioValue">Object A</KRadio>
+  <KRadio name="test" :selected-value="objB" v-model="radioValue">Object B</KRadio>
 </template>
 
-<script lang="ts">
-import { defineComponent, reactive, toRefs } from 'vue'
-
-export default defineComponent({
-  setup() {
-    const data = reactive({
-      objA: { name: 'a' },
-      objB: { name: 'b' },
-      radioGroup: 'string',
-    })
-
-    return {
-      ...toRefs(data),
-    }
-  }
-})
+<script setup lang="ts">
+import { ref } from 'vue'
+
+const radioValue = ref<boolean | string | object>('string')
+
+const objA = { name: 'a' }
+const objB = { name: 'b' }
 </script>
 ```
 
 ## Props
 
-### v-model - required
+### v-model
 
-Use `v-model` to bind the `checked` state of the underlying `<input />`. The `v-model` binds to the `modelValue` prop of the component and sets the current checked state of the input. You can read more about passing values via `v-model` [here](https://vuejs.org/guide/components/events.html#usage-with-v-model).
+Use `v-model` to bind the `checked` state of the underlying `<input>` element. The `v-model` binds to the `modelValue` prop of the component and sets the current checked state of the input. You can read more about passing values via `v-model` [here](https://vuejs.org/guide/components/events.html#usage-with-v-model).
 
-### selectedValue - required
+### selectedValue
 
-The value of the `KRadio` option that will be emitted by the `change` and `update:modelValue` events.
+The value of the KRadio option that will be emitted by the `change` and `update:modelValue` events.
 
 ### label
 
 Will place label text to the right of the radio. Can also be [slotted](#slots).
 
-<KRadio v-model="checked" label="Label Example" :selected-value="true" />
+<KRadio v-model="labelPropRadio" label="Label example" :selected-value="true" />
```

Comment:
The label and the radio don't appear to be vertically aligned. The label text appears slightly lower than the radio button

![image](https://github.com/Kong/kongponents/assets/2229946/15a55770-2dd9-4e24-b260-21f6b62479a0)

Reply:
All instances
![image](https://github.com/Kong/kongponents/assets/2229946/18c35cd3-4727-4c3f-881e-c70908760921)